var/spack/repos/builtin/packages/root/package.py

```diff
@@ -110,6 +112,9 @@ class Root(CMakePackage):
     # otherwise it crashes with the internal minuit library
     variant('minuit', default=True,
             description='Automatically search for support libraries')
+    variant('mlp', default=False,
+            description='Enable support for TMultilayerPerceptron '
+            'classes\' federation')
```

Comment:
If you use double quotes as your string delimiter you won’t need to escape the apostrophe

Reply:
done.